samwise.js

```diff
@@ -1,20 +1,19 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.